src/hooks/useSnackMessage.ts

```diff
@@ -89,14 +91,12 @@ function makeSnackbar(
         snackInputs.headerId,
         snackInputs.headerValues
     );
+
     if (message !== null && header !== null) {
-        displayMessageWithSnackbar(
-            message,
-            header,
-            enqueueSnackbar,
-            level,
-            persistent
-        );
+        displayMessageWithSnackbar(message, header, enqueueSnackbar, level, {
+            key: snackInputs.key,
+            persist: snackInputs.persist,
+        });
```

Comment:
You could pass snackInputs directly, to make sure that if we want additional properties we can still pass them

Reply:
snackInputs contains others properties that are not needed for enqueueSnackbar, that why i extracted the required one.
but i suppose enqueueSnackbar don't care if we give it more than what it needs